src/OrchardCore.Modules/OrchardCore.Contents/Razor/ContentRazorHelperExtensions.cs

```diff
@@ -46,26 +46,26 @@ public static async Task<ContentItem> GetContentItemByHandleAsync(this IOrchardH
     /// </summary>
     /// <param name="orchardHelper">The <see cref="IOrchardHelper"/>.</param>
     /// <param name="contentItemId">The content item id to load.</param>
-    /// <param name="latest">Whether a draft should be loaded if available. <c>false</c> by default.</param>
+    /// <param name="option">The version to load. By default we load the Published version.</param>
```

Comment:
nit: don't use we/I in comments. Use past tense ("published version is loaded")

Reply:
@sebastienros updated.